src/interfaces/colors.ts

```diff
@@ -0,0 +1,12 @@
+export enum IColors {
```

Comment:
What does `interfaces` name refer to?

If it refers to ui elements, I suggest we rename it to `ui` or even move to `style`, to avoid confusion with interface _as a contract between modules_.

If it refers to a module grouping common types, I suggest with move these types to module more scoped around a specific context.